### DISCUSSION TITLE
p/de4b9576/

### DISCUSSION THREAD

**Discussion by giscus** [MHuiG/talk#10] [Giscus]
# p/de4b9576/

重装系统前的备份列表 以下内容适用于 win10 与 win11 - MHuiG - MHuiG

https://blog.mhuig.top/p/de4b9576/

**Comment by brucmao** [comment]
好全，收藏了

重装可以用rufus 自作启动盘时有选项可以禁用 tpm 和 在线登录用户
![](https://s3.bmp.ovh/imgs/2023/01/01/b48cf52a2196f2da.png)

软件安装我喜欢用scoop ,[重装恢复](https://jiayaoo3o.github.io/2019/03/19/%E9%87%8D%E8%A3%85%E7%B3%BB%E7%BB%9F%E5%90%8E%E5%A6%82%E4%BD%95%E6%81%A2%E5%A4%8D%E4%BD%BF%E7%94%A8scoop/)也方便

不过今天重装win11遇到一个问题，重装后发现一个文件夹下ini文件无法修改，发现是因为文件夹所有者还是重装之前用户对应的GUID,将整个文件夹权限所有者改为当前登录用户（我一直用的微软在线账号登录），并勾选替换子容器和对象的所有者，应用之后还是无法编辑。最后只能采用建立同名文件夹，把文件全部移动过去，才能修改。

win重装之后，其他盘的文件所有者都是S-1-5-21-1279403538 这样的一串数字，批量改还挺麻烦的。查了下好像可以使用icacls 命令批量修改。
